docs/additional-functionality/rapids-udfs.md

```diff
@@ -134,29 +134,28 @@ implements a Hive simple UDF using
 [native code](../../udf-examples/src/main/cpp/src) to count words in strings
 
 
-## GPU Scheduling For Pandas UDF
+## GPU support for Pandas UDF
 
 ---
 **NOTE**
 
-The _GPU Scheduling for Pandas UDF_ is an experimental feature, and may change at any point it time.
+The _GPU support for Pandas UDF_ is an experimental feature, and may change at any point it time.
 
 ---
 
-_GPU Scheduling for Pandas UDF_ is built on Apache Spark's [Pandas UDF(user defined
-function)](https://spark.apache.org/docs/3.0.0/sql-pyspark-pandas-with-arrow.html#pandas-udfs-aka-vectorized-udfs),
-and has two components:
+_GPU support for Pandas UDF_ is built on Apache Spark's [Pandas UDF(user defined
+function)](https://spark.apache.org/docs/latest/api/python/user_guide/arrow_pandas.html#pandas-udfs-a-k-a-vectorized-udfs),
+and has two features:
 
-- **Share GPU with JVM**: Let the Python process share JVM GPU. The Python process could run on the
-  same GPU with JVM.
+- **GPU Assignment(Scheduling) in Python Process**: Let the Python process share the same GPU with Spark executor JVM. Without this feature, some use case in PandasUDF(a.k.a an `independent` process) will likely to use other GPUs other than the one we want it to run on. e.g. user can launch a TensorFlow session inside Pandas UDF and the machine contains 8 GPUs. user launchs 8 Spark executors. Without this GPU sharing feature, TensorFlow will automatically use all 8 GPUs it can detects which will definitly conflict with existing Spark executor JVM processes.
```

Comment:
Please wrap lines to approx. 100 characters to increase readability when examining the documentation in a terminal and to make editing/commenting easier.  Edits to this really long line below:
```suggestion
- **GPU Assignment(Scheduling) in Python Process**: Lets the Python process share the same GPU
with the Spark executor JVM process. Without this feature, GPU usage in a Pandas UDF could use
a different GPU than the one used by the Spark executor.  As an example, consider a TensorFlow
session launched inside a Pandas UDF and the machine contains 8 GPUs. The user launches 8 Spark
executors. Without this GPU sharing feature, TensorFlow will automatically use all 8 GPUs it detects
which will conflict with the Spark executor JVM processes.
```

Reply:
Thanks for this point! Updated.